content/desktop/install/msi/faq.md

```diff
@@ -0,0 +1,37 @@
+---
+title: MSi FAQs
```

Comment:
From the doc, I have a section called [Formatting JSON properties](https://www.notion.so/dockerinc/The-admins-guide-to-installing-Docker-Desktop-871b4e14ea464d83a31229883d72ad3d?pvs=4#57cc7b2256934566b8b73f4873560093).

Do you think it would work in FAQs even though it's more of a "tips and tricks" section?

Reply:
Sounds good to me. Thank you!